### PR TITLE
Certificates two

### PR DESCRIPTION
**Pull request by Mudassar045**

### CODE REVIEW

File: web/src/components/Layout/index.js [hunk 1]
```diff
@@ -25,13 +25,17 @@ const Header = ({user, history}) => <div className="header">
...
-				<div className="logo-container" style={{width: "20%"}}>
-					{logo !== "" && <img className="header-logo" src={logo} alt="No Logo"/>}
+				<div className="logo-container" style={{width: "20%", objectFit:"center"}}>
+					{logo !== "" && <img className="header-logo" src={logo} alt="School Logo"/>}
 				</div>
 				<div className="header-style">
-					<div className="title">{settings.schoolName}</div>
-					<div className="address" style={{marginBottom: "5px"}}>{settings.schoolAddress}</div>
-					<div className="phone-number">{settings.schoolPhoneNumber}</div>
+					<div className="title">{settings.schoolName.toUpperCase() || ""}</div>
```
Comment (taimur38) [inline web/src/components/Layout/index.js]: this doesn't handle the case if schoolName is undefined.

File: web/src/components/Layout/index.js [hunk 2]
```diff
@@ -25,13 +25,17 @@ const Header = ({user, history}) => <div className="header">
 
 export const PrintHeader = ({settings, logo}) => <div className="print-only school-header">
 			<div className="header-body">
-				<div className="logo-container" style={{width: "20%"}}>
-					{logo !== "" && <img className="header-logo" src={logo} alt="No Logo"/>}
+				<div className="logo-container" style={{width: "20%", objectFit:"center"}}>
```
Comment (taimur38) [inline web/src/components/Layout/index.js]: don't think objectFit is needed here

File: web/src/modules/Student/List/index.js [hunk 3]
```diff
@@ -29,6 +29,11 @@ const StudentItem = (S) => {
 						<div className="para-row">
 							<b>{`Adm #: `}</b>{S.AdmissionNumber}
 						</div>}
+					{
+						<div className="para-row">
+							<b>{`Phone:`}</b>{S.Phone?S.Phone.substr(0,10):""}
```
Comment (taimur38) [inline web/src/modules/Student/List/index.js]: put spaces in between these commands. 
` S.Phone ? S.Phone.substr(0, 10) : ""`

also why do we need this substring.

File: web/src/modules/Student/Single/Certificates/style.css [hunk 4]
```diff
@@ -101,9 +101,20 @@
     @media print {
         .certificate {
             margin-top: 50px;
-            border: 10px double rgba(0,0,0,0.3);
```
Comment (taimur38) [inline web/src/modules/Student/Single/Certificates/style.css]: it should have a border. almost every example certificate had a border around it

File: web/src/components/Layout/index.js (hunk 1 above)
Comment (Mudassar045) [reply web/src/components/Layout/index.js]: Acknowledged, I'll apply validations here

File: web/src/components/Layout/index.js (hunk 2 above)
Comment (Mudassar045) [reply web/src/components/Layout/index.js]: ok, I will remove it

**Pull request merged**